### ISSUE TITLE
Code Tabs are not accessible; fail Deque Axe plugin test

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA#745]
We use tabbed layouts in places where we want to display source code. The tabs allow us to show implementations of algorithms and data structures in multiple languages, and then use code mirror to apply syntax coloring. 

The tabs that are displayed are [jQuery UI widgets](https://jqueryui.com/tabs/). We built an extension that looks for `:codeinclude:` directives, along with a name for the source file(s) to be displayed. This extension is implemented in `RST/ODSAextensions/odsa/codeinclude/codeinclude.py`, but relies on JQuery UI to draw them on screen.

There are two main issues:

- The blue tab/white text in the active tab has a color contrast that is too low. This blue is set in `jQuery.ui.min.css`. We can change the color in their files, but it won't be sticky across JQuery updates. Ideally, we would override the tab CSS in our CSS files. The color #0066cc maintains the blue background but is a bit darker, improving the contrast ratio to the white text.

- The tabs themselves generate the following error in the AXE tool: `Ensure interactive controls are not nested as they are not always announced by screen readers or can cause focus problems for assistive technologies` See [the Deque site](https://dequeuniversity.com/rules/axe/4.11/nested-interactive?application=AxeChrome) for more info on this error. 

We may need to update jquery UI from 1.21.1 to latest to fix the nested control issue, as we have no control over how the the tabs are actually rendered. Updating to jQuery latest will likely require an update to a later version of jQuery. We are currently using 3.5.5, and we would need to go to at least ver 3.7.1. There is a version 4.0, but it just came out, so not sure about stability.

Alternatively, we can dispense with jQuery UI for the tabs (or maybe altogether?), and move to another UI library. I'm not sure how much we use jquery/jquery UI in the rest of the app, so this might prove to be ill-advised.

**Comment by cashaffer** [comment]
This should be revisited now that we have updated jQuery and jQueryUI.
